### ISSUE TITLE
Camera not found when launched with robot_upstart

### ISSUE THREAD

**Issue by CaptD** [KumarRobotics/flea3#19]
Hi,

I am developing a robotic function using the PointGrey Chameleon camera, and now I want to open the camera automatically when the computer boots using robot_upstart library (http://docs.ros.org/jade/api/robot_upstart/html/index.html).

However, the camera won't boot, and the error type is PGRERROR_NOT_FOUND, catched in flea3_camera.cpp. I checked udev rule to be MODE="0777", and also tried delay the roslaunch for 10 seconds, but neither worked...

Could you help to give some advice?

Thanks,
Jin

**Comment by versatran01** [comment]
Does  it work when you manually launch it?

**Comment by CaptD** [comment]
Yes, the camera works very well when launched manually after the computer starts up.
